### PR TITLE
add torchani

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/torchani/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{% set name = "torchani" %}
+{% set version = "2.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  git_url: git+https://github.com/aiqm/torchani.git#egg={{ version }}
```
Comment (dopplershift) [inline recipes/torchani/meta.yaml:10]: Why use the git URL rather than the tarball up on PyPI?

Comment (hadim) [reply recipes/torchani/meta.yaml:10]: I tried but the source tarball is messed up and not standard so install fails. I also tried the source dir from GH release but the installation failed too.

Comment (hadim) [reply recipes/torchani/meta.yaml:10]: Issue has been reported upstream https://github.com/aiqm/torchani/issues/507

**Comment by hadim** [comment]
LGTM @dopplershift

**Pull request merged**